masterlist.yaml

```
@@ -514,6 +514,14 @@ plugins:
         subs: [ '"Script Improvements" version' ]
         condition: 'many("gr_ScriptImprovements.es(m|p)")'
 
+###### Environment, Landscape & Flora ######
+  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
```

Add the location as well.
```suggestion
  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49464/' ]
```